runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -919,4 +927,25 @@ private int ping(World world)
 		storedPings.put(world.getId(), ping);
 		return ping;
 	}
+
+	private Map<Integer, Boolean> getIsWestCoastWorld()
+	{
+		if (isWestCoastWorld == null)
+		{
+			// Map worlds to whether or not they are East/West coast
+			// https://abextm.github.io/cache2/#/viewer/enum/4992
+			// https://oldschool.runescape.wiki/w/Server
+			// As of writing this the runelite API doesn't give world coast information: https://api.runelite.net/runelite-1.10.18-SNAPSHOT/worlds.js
+			final EnumComposition worldCoastEnum = client.getEnum(EnumID.WORLD_COASTS);
```

Comment:
This will crash the client. You can see this and other api misuse by enabline assertions.

Reply:
Updated to be run on the client thread.